pkg/ipcache/metadata.go

```diff
@@ -544,29 +541,29 @@ func sourceByLabels(d source.Source, lbls labels.Labels) source.Source {
 // The following diagram describes the relationship between the label injector
 // triggered here and the callers/callees.
 //
-//      +------------+  (1)        (1)  +-----------------------------+
-//      | EP Watcher +-----+      +-----+ CN Watcher / Node Discovery |
-//      +-----+------+   W |      | W   +------+----------------------+
-//            |            |      |            |
-//            |            v      v            |
-//            |            +------+            |
-//            |            | IDMD |            |
-//            |            +------+            |
-//            |               ^                |
-//            |               |                |
-//            |           (3) |R               |
-//            | (2)    +------+--------+   (2) |
-//            +------->|Label Injector |<------+
-//           Trigger   +-------+-------+ Trigger
-//                         (4) |W
-//                             |
-//                             v
-//                           +---+
-//                           |IPC|
-//                           +---+
-//      legend:
-//      * W means write
-//      * R means read
+//	+------------+  (1)        (1)  +-----------------------------+
+//	| EP Watcher +-----+      +-----+ CN Watcher / Node Discovery |
+//	+-----+------+   W |      | W   +------+----------------------+
+//	      |            |      |            |
+//	      |            v      v            |
+//	      |            +------+            |
+//	      |            | IDMD |            |
+//	      |            +------+            |
+//	      |               ^                |
+//	      |               |                |
+//	      |           (3) |R               |
+//	      | (2)    +------+--------+   (2) |
+//	      +------->|Label Injector |<------+
+//	     Trigger   +-------+-------+ Trigger
+//	                   (4) |W
+//	                       |
+//	                       v
+//	                     +---+
+//	                     |IPC|
+//	                     +---+
+//	legend:
+//	* W means write
+//	* R means read
```

Comment:
These white space updates weren't part of the upstream commit. Not sure if we really should backport them.

Reply:
thanks, it could be auto format from IDE, let me revert these unintended changes.